### PR TITLE
Fix outdated upstream doc on user access

### PR DESCRIPTION
**Pull request by jotak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,32 +175,39 @@ More performance fine-tuning is possible when using Kafka, ie. with `spec.deploy
...
-  verbs:
-  - 'get'
+```bash
+oc adm policy add-cluster-role-to-user netobserv-reader test
 ```
 
-- per-app auth, ie. with NetObserv components having access to the logs using their own service account, regardless of the user permissions. For this mode, set `spec.loki.authToken` to `HOST` and setup `ClusterRole` for `flowlogs-pipeline` and `netobserv-plugin`, e.g: [role.yaml](https://github.com/netobserv/documents/blob/main/examples/loki-stack/role.yaml).
+More information about multi-tenancy can be found on [this page](https://github.com/netobserv/documents/blob/main/loki_operator.md#netobserv-configuration).
+
+Note that multi-tenancy is not possible when not using the Loki Operator.
```
Comment (jpinsonneau) [inline README.md:196]:
```suggestion
Note that multi-tenancy is not possible without using the Loki Operator.
```

File: README.md [hunk 2]
```diff
@@ -175,32 +175,39 @@ More performance fine-tuning is possible when using Kafka, ie. with `spec.deploy
...
+
+#### Network Policy
 
+For a production deployment, it is also highly recommended to lock down the `netobserv` namespace (or wherever NetObserv is installed) using network policies.
+An example of such a network policy is [provided here](https://github.com/netobserv/documents/blob/main/examples/lockdown-netobserv.yaml).
```
Comment (jpinsonneau) [inline README.md:201]:
```suggestion
An example of network policy is [provided here](https://github.com/netobserv/documents/blob/main/examples/lockdown-netobserv.yaml).
```

File: README.md [hunk 3]
```diff
@@ -175,32 +175,39 @@ More performance fine-tuning is possible when using Kafka, ie. with `spec.deploy
...
+
+#### Communications
+
+By default, communications between internal components are not secured. Note that, when using the Loki Operator, securing communication with TLS is necessary. There are several places where TLS can be set up:
+
+- Connections to Loki (from the processor `flowlogs-pipeline` and from the Console plugin), by setting `spec.loki.tls`.
+- With Kafka (both on producer and consumer sides), by setting `spec.kafka.tls`. Mutual TLS is supported here.
+- The metrics server running in the processor (`flowlogs-pipeline`) can listen using TLS, via `spec.processor.metrics.server.tls`.
```
Comment (jpinsonneau) [inline README.md:209]:
```suggestion
- The metrics server (running in the processor `flowlogs-pipeline`) can listen using TLS, via `spec.processor.metrics.server.tls`.
```

File: README.md [hunk 4]
```diff
@@ -175,32 +175,39 @@ More performance fine-tuning is possible when using Kafka, ie. with `spec.deploy
...
+NetObserv is meant to be used by cluster admins, or, when using the Loki Operator (v5.7 or above), project admins (ie. users having admin permissions on some namespaces only). Multi-tenancy is based on namespaces permissions, with allowed users able to get flows limited to their namespaces. Flows across two namespaces will be visible to them as long as they have access to at least one of these namespaces.
 
-- Between processor (`flowlogs-pipeline`) and Loki, by setting `spec.loki.tls`.
-- With Kafka (both on producer and consumer sides), by setting `spec.kafka.tls`. Mutual TLS is supported here.
-- The metrics server running in the processor (`flowlogs-pipeline`) can listen using TLS, via `spec.processor.metrics.server.tls`.
-- The Console plugin server always uses TLS.
+To make autorized queries to Loki using the Loki Operator, NetObserv must be configured as such:
 
-To make authenticated queries to Loki, when using the Loki Operator, there are two options:
+- set `spec.loki.authToken` to `FORWARD` in the `FlowCollector` resource. The console plugin will forward the logged-in Console user token to Loki.
```
Comment (jpinsonneau) [inline README.md:184]:
```suggestion
- set `spec.loki.authToken` to `FORWARD` in the `FlowCollector` resource. The console plugin will forward the logged-in [OCP Console](https://github.com/openshift/console) user token to Loki [Gateway API](https://github.com/observatorium/api).
```

**Review by jpinsonneau — DISMISSED** [review]
LGTM, just small suggestions. Feel free to pick some on these :+1:

File: README.md (hunk 3 above)
Comment (jotak) [reply README.md:209]: thanks @jpinsonneau 
I didn't take that one: there's several metrics servers (in flp but also in console plugin and in the operator) so I think parenthesis should be kept just around "flowlogs-pipeline"

**Pull request merged**